include/constants/trainer_hill.h

```diff
@@ -0,0 +1,29 @@
+#ifndef GUARD_CONSTANTS_TRAINER_HILL_H
+#define GUARD_CONSTANTS_TRAINER_HILL_H
+
+#define TRAINER_HILL_FUNC_START 0
```

Comment:
I think this is fine for now.   In the future, it might be nice to create macros for the various battle frontier "CallXFunction" specials.